### PR TITLE
Csv logger docs

### PR DESCRIPTION
**Pull request by KeremTurgutlu**
I've fixed a minor typo in callback and added docs.

### CODE REVIEW

File: docs_src/callbacks.csv_logger.ipynb [hunk 1]
```diff
@@ -32,7 +32,7 @@
       "text/markdown": [
```
Comment (sgugger) [inline docs_src/callbacks.csv_logger.ipynb]: <p>There is a weird warning here, can you get rid of it?</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1833/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (sgugger) [inline docs_src/callbacks.csv_logger.ipynb]: <p>Can you use the path variable here, with the function `os.remove` maybe? Notebooks are automatically executed as parts of our tests, so all the cells need to be able to execute anywhere.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1833/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by sgugger** [comment]
Thanks a lot! There are just two minors things to change in the notebook before we can merge, if possible.

File: docs_src/callbacks.csv_logger.ipynb (hunk 1 above)
Comment (KeremTurgutlu) [reply docs_src/callbacks.csv_logger.ipynb]: <p>Sure will do that!</p>

Comment (KeremTurgutlu) [reply docs_src/callbacks.csv_logger.ipynb]: <p>Yeap, let me look into that, thanks :)</p>

Comment (KeremTurgutlu) [reply docs_src/callbacks.csv_logger.ipynb]: <p>Issue seems to be addressed here https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation#how-to-disable-the-warning. But it's weird if I am the only one getting it or is it related to 		`pyyaml` version?</p>

Comment (sgugger) [reply docs_src/callbacks.csv_logger.ipynb]: I don't have it, personally.

**Comment by sgugger** [comment]
Looks good now, thanks!

**Pull request merged**